### DISCUSSION TITLE
How to create Blazorise Bootstrap Demo theme in Blazor server App?

### DISCUSSION THREAD

**Discussion by imhemayatsangin** [Megabit/Blazorise#2638] [General]
I'm going to add the Blazorise Bootstrap Demo theme which is shown in the demo URL, I'm using Blazor Server App, i tried and copied all the source file but i'm not able to create one like the demo, Please if anybody suggest me or guide me how to create it with all the steps?

**Comment by stsrki** [comment]
Demo for **server app** is hosted in https://github.com/Megabit/Blazorise/tree/master/Demos/Blazorise.Demo.Bootstrap.RC

and the actual pages and layout that is shared across all demo apps is located in https://github.com/Megabit/Blazorise/tree/master/Demos/Blazorise.Demo

**Reply by imhemayatsangin** [reply]
Thanks dear for your prompt and useful answer,  solved it with your answer. Appreciated.

**Reply by stsrki** [reply]
Np. Can you please accept the answer? :)
